smart-contracts/package.json

```diff
@@ -33,6 +33,7 @@
     "unlock-abi-0-1": "1.0.0",
     "unlock-abi-0-2": "1.0.0",
     "unlock-abi-1-0": "1.0.0",
+    "unlock-abi-1-1": "^1.0.0",
```

Comment:
I could do this in a separate pr and then rebase. This was already in `node_modules` which is why I missed it, but CI is complaining that it needs to be in our `package.json`.

Reply:
Should pin that as well (i.e. remove the `^`)

Reply:
Right. I've just done this in a separate PR.